### PR TITLE
Added SonarQube support 

### PR DESCRIPTION
**Pull request by nicolas63**

### CODE REVIEW

**Comment by danielpetisme** [comment]
so cool!!! I was expecting sonar for so long!!!

can you have a look to th ci result.
I can see linting issues.

thanks

File: README.md [hunk 1]
```diff
@@ -104,6 +104,31 @@ docker run -d -p [A host port]:80 [Image name]:[Image tag]
...
+
+## Manually : 
+
+1. Run Sonar in container : ```docker-compose -f ./docker/sonar.yml up -d```
+
+2. Install sonar scanner for .net :
+   
+ ```dotnet tool install --global dotnet-sonarscanner```
+
+3. Run ```dotnet sonarscanner begin /d:sonar.login=admin /d:sonar.password=admin /k:"AwesomeKey" /d:sonar.host.url="http://localhost:9001"```
```
Comment (danielpetisme) [inline README.md]: can we unify the way sonar is triggered either by container or manually? Use the ps1 script in both places ? or in none? having a cli to copy/paste from the Readme.md is good enough

File: generators/common/index.js [hunk 2]
```diff
@@ -36,6 +36,8 @@ module.exports = class extends CommonGenerator {
         this.configOptions = jhContext.configOptions || {};
     }
 
+
```
Comment (danielpetisme) [inline generators/common/index.js]: 1 empty line should be enough

File: README.md (hunk 1 above)
Comment (nicolas63) [reply README.md]: In the two cases i run the docker and the ps1 script only launches the 4 commands that can be seen in the manual section

**Comment by nicolas63** [comment]
fix #9

**Comment by nicolas63** [comment]
Bounty claimed https://opencollective.com/generator-jhipster/expenses/14160

**Comment by pascalgrimaud** [comment]
just approuved :)

FYI, you need to add 'fix #...' in the first comment to close the ticket, not in other comment

**Pull request merged**